webapi/webapi-contactsmanager-w3c-tests/contactsmanager/Contact.html

```diff
@@ -42,7 +42,7 @@
 
 var contact;
 var data = new Date(1987,10,27);
-var dataString = data.toString();
+var dataTime = data.getTime();
```

Comment:
var dataTime = data;
The `birthday` is a `Date` type, refers to http://www.w3.org/2012/sysapps/contacts-manager-api/#idl-def-Contact